api/src/pcapi/utils/csr.py

```diff
@@ -10,14 +10,14 @@ class Csr(TypedDict):
     csr_id: str
 
 
-def get_closest_csr(gtl_id: str) -> Csr | None:
+def get_closest_csr(gtl_id: str, **kwargs) -> Csr | None:
     gtl_id = gtl_id.zfill(8)
     csr_level_1 = get_csr(f"{gtl_id[:2]}000000")
     csr_level_2 = get_csr(f"{gtl_id[:4]}0000")
     csr_level_3 = get_csr(f"{gtl_id[:6]}00")
     csr_level_4 = get_csr(gtl_id)
     csr = csr_level_4 or csr_level_3 or csr_level_2 or csr_level_1
-    if csr is None:
+    if csr is None and "logging" in kwargs and kwargs["logging"] is not False:
```

Comment:
```suggestion
    if csr is None and "logging" in kwargs and kwargs["logging"]:
```
C'est suffisant non ?

Reply:
Et peut-être que 
```suggestion
    if csr is None and kwargs.get("logging"):
```
marche aussi, mais je suis pas 100% certain